_topic_map.yml

```diff
@@ -1627,6 +1627,9 @@ Topics:
 # Release notes
 - Name: Release Notes
   File: serverless-release-notes
+# Support
+# - Name: Support
+#  File: serverless-support
```

Comment:
Why is this commented out? A temporary change?

Reply:
I'm curious if you plan to rename the section to "Support" or something else.

Reply:
The file is called `serverless-support` . Not sure where the file is but the original file is still called `serverless-getting-support.adoc`. Maybe it should be renamed?